apex-app-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -50,6 +50,12 @@
       </includes>
     </fileSet>
     <fileSet filtered="true" encoding="UTF-8">
+      <directory>src/main/resources</directory>
+      <includes>
+        <include>**/apexlog4j.properties</include>
```

Comment:
After revisiting here are my findings:
I would prefer to name this file differently so that for users it's clear that a custom logger cong file is used. As hadoop and other logger config files are likely to have name log4j.properties, sometimes it could be confusing to figure out which one is getting loaded currently.
Technically, it's little tricky as "/home/hduser/datatorrent/releases/3.7.0-SNAPSHOT/conf" is in classpath when ApexCli is launched and if we have file named log4j.properties there, it will be picked and the ApexCli will try to use to use our logger configuration and hence will redirect logs to a file rather than console. Even if I manage to fix this place, I am sure it will create such confusions at some place or other.
That's why I feel it's good to give it distinct name so we can selectively use it, when we really want to. And we don't end up loading it accidently.